Formula/semgrep.rb

```diff
@@ -4,8 +4,8 @@ class Semgrep < Formula
   desc "Easily detect and prevent bugs and anti-patterns in your codebase"
   homepage "https://semgrep.dev"
   url "https://github.com/returntocorp/semgrep.git",
-    tag:      "v0.19.1",
-    revision: "e9e42cf838790741bda870735b942cc736b80fd2"
+    tag:      "v0.20.0",
+    revision: "1e89dcc5d1e88d447397b663d5de7ce2a38976c0"
   license "LGPL-2.1"
```

Comment:
```suggestion
  license "LGPL-2.1-only"
```
to get rid of deprecated license label